website/thaliedje/templates/thaliedje/search.html

```diff
@@ -4,11 +4,14 @@
 <div id="player-search-container-{{ player.id }}">
     <input v-model="query" id="search" type="text" maxlength="100" placeholder="Search for something to play" class="container-fluid form-control mb-3"/>
     <div id="results" class="px-3 search-list">
-        <div class="d-flex flex-column">
-            <template v-if="tracks.length > 0 && query !== ''">
+        <div class="row">
+            <div class="col-lg-4" v-if="tracks.length > 0 && query !== ''">
                 <h3>Tracks</h3>
                 <template v-for="(track, index) in tracks">
                     <div class="d-flex my-2">
```

Comment:
Maybe try making this whole div clickable for adding songs, perhaps by adding `strechted-link` to the add button (https://getbootstrap.com/docs/5.2/helpers/stretched-link/)

Reply:
Let's make an issue for this